### PR TITLE
refactor(FOROME-955): get rid of styled components

### PR DESCRIPTION
**Pull request by EvgeniySobolev**
[https://quantori.atlassian.net/browse/FOROME-955](https://quantori.atlassian.net/browse/FOROME-955)

### CODE REVIEW

File: src/pages/filter/dtree/components/query-builder/ui/next-step/all-mode-wrapper/all-mode-wrapper.module.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.all-mode-wrapper {
+  margin: 8px 4px;
+  width: 25px;
+  height: 20px;
+  color: theme("colors.green.secondary");
+  background-color: theme("colors.green.medium");
+  font-size: 12px;
```
Comment (Lapk1n) [inline src/pages/filter/dtree/components/query-builder/ui/next-step/all-mode-wrapper/all-mode-wrapper.module.css]: theme('fontSize')

File: src/index.css [hunk 2]
```diff
@@ -80,3 +80,49 @@
...
+
+/* stylelint-disable-next-line selector-class-pattern */
+.Toastify__toast {
+  max-height: 52px;
+  min-height: auto;
+}
+
+/* stylelint-disable-next-line selector-class-pattern */
+.Toastify__toast-body {
+  font-size: 14px;
```
Comment (Lapk1n) [inline src/index.css]: theme('fontSize')

File: src/index.css [hunk 3]
```diff
@@ -80,3 +80,49 @@
...
+}
+
+.react-switch-handle {
+  width: 12px;
+  height: 12px;
+}
+
+/* stylelint-disable-next-line selector-class-pattern */
+.Toastify__toast--info {
+  background-color: #18a0fb;
```
Comment (Lapk1n) [inline src/index.css]: theme('colors')

File: src/index.css [hunk 4]
```diff
@@ -80,3 +80,49 @@
...
+/* stylelint-disable-next-line selector-class-pattern */
+.Toastify__toast {
+  max-height: 52px;
+  min-height: auto;
+}
+
+/* stylelint-disable-next-line selector-class-pattern */
+.Toastify__toast-body {
+  font-size: 14px;
+  line-height: 16px;
```
Comment (Lapk1n) [inline src/index.css]: theme('lineHeight')

File: src/pages/filter/dtree/components/query-builder/ui/next-step/next-step-route/next-step-route.module.css [hunk 5]
```diff
@@ -0,0 +1,63 @@
+.next-step-route {
+  position: relative;
+  display: flex;
+  height: 100%;
+  width: 100%;
+
+  &__start-amount {
+    font-size: 13px;
```
Comment (Lapk1n) [inline src/pages/filter/dtree/components/query-builder/ui/next-step/next-step-route/next-step-route.module.css]: theme('fontSize')

File: src/pages/filter/dtree/components/query-builder/ui/next-step/not-mode-wrapper/not-mode-wrapper.module.css [hunk 6]
```diff
@@ -0,0 +1,8 @@
+.not-mode-wrapper {
+  margin: 8px 4px;
+  width: 25px;
+  height: 20px;
+  color: theme("colors.red.light");
+  background-color: theme("colors.red.lighter");
+  font-size: 12px;
```
Comment (Lapk1n) [inline src/pages/filter/dtree/components/query-builder/ui/next-step/not-mode-wrapper/not-mode-wrapper.module.css]: theme('fontSize')

File: src/pages/filter/dtree/components/query-builder/ui/next-step/not-mode-wrapper/not-mode-wrapper.tsx [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+interface INotModeWrapperProps {
+  className?: Argument
+  children: ReactNode
+}
+
+export const NotModeWrapper = ({
+  className,
+  children,
+}: INotModeWrapperProps): ReactElement => {
+  return <div className={cn(styles.notModeWrapper, className)}>{children}</div>
```
Comment (Lapk1n) [inline src/pages/filter/dtree/components/query-builder/ui/next-step/not-mode-wrapper/not-mode-wrapper.tsx]: No need for return

File: src/pages/not-found/not-found.module.css [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+  &__title {
+    font-weight: 500;
+    font-size: 36px;
+    line-height: 42px;
+    color: theme("colors.blue.dark");
+  }
+
+  &__info {
+    font-size: 16px;
```
Comment (Lapk1n) [inline src/pages/not-found/not-found.module.css]: theme('fontSize'). Check other similar cases too

File: src/pages/not-found/not-found.module.css [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+    line-height: 24px;
```
Comment (Lapk1n) [inline src/pages/not-found/not-found.module.css]: theme('lineHeight') Check other similar cases too

**Review by Lapk1n — COMMENTED** [review]
Screenshot 2022-08-11 at 11 11 41
Lo
Screenshot 2022-08-11 at 11 11 59
It looks like there is some difference with the slider view in the maker

**Comment by AndreyUstyumenko** [comment]
@Lapk1n 
Num_samples works the same on QA. It's a feature.
Screen Shot 2022-08-11 at 19 23 13

**Review by AndreyUstyumenko — APPROVED** [review]

**Review by Lapk1n — APPROVED** [review]

**Pull request merged**